frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -33,10 +33,18 @@ export const LSO_FLAG = 'LSO';
 
 export const RGW_FLAG = 'RGW';
 
-/* Key should be same as values received from the CSV 
-  Values should be formatted as OCS_<Console-Flag-Name> */
-export const OCS_SUPPORT_FLAGS = {
-  MULTUS: 'OCS_MULTUS',
+export enum FEATURES {
```

Comment:
Please add a comment on how to relate the two and which one should contain values from CSV.